srtcore/buffer_rcv.h

```diff
@@ -107,7 +345,7 @@ class CRcvBuffer
     /// @return actual number of bytes extracted from the buffer.
     ///          0 if nothing to read.
     ///         -1 on failure.
-    int readMessage(char* data, size_t len, SRT_MSGCTRL* msgctrl = NULL);
+    int readMessage(char* data, size_t len, SRT_MSGCTRL* msgctrl = NULL, std::pair<int32_t, int32_t>* pw_seqrange = NULL);
```

Comment:
Update doxigen comment with the added argument.

Reply:
Will do separately, I can't force git to mark the right region to fix ;)